db_stress_tool/db_stress_test_base.cc

```diff
@@ -1592,7 +1592,7 @@ Status StressTest::TestIterateImpl(ThreadState* thread,
     ro.total_order_seek = true;
     expect_total_order = true;
   } else if (thread->rand.OneIn(4)) {
-    ro.total_order_seek = false;
+    ro.total_order_seek = thread->rand.OneIn(2);
```

Comment:
We would want to add the new option to the stress/crash test framework as well

Reply:
It looks tricky to add because there's a surprising lot of stress test code doing prefix iteration. `prefix_seek_opt_in_only` is not unlocking any new functionality. It's just a cosmetic shortcut for forcing total_order_seek.